src/python/Makefile

```diff
@@ -16,7 +16,6 @@ CODEDIR := ../src/python/foglamp
 check-root:
 ifeq ($(USER), root)
 	@echo "WARNING: Installing as root should be avoided. Use a virtual environment."
```

Comment:
We are moving away from virtualenv

Reply:
Fixed the text, however kept the check (removed exit already, if root)